test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -61,6 +65,40 @@ var removeBubbleListener = function (eventType) {
     document.removeEventListener(eventType, bubbleEventListener);
 };
 
+test('mouse click event detail value', function () {
```

Comment:
my proposal about test structuring:

```js
module(e.detail);

test('click');
test('doubleClick');
test('mouseDown');
...
```
It'll be more compact and clear naming without a lot of redundant words (`mouse mouse down event detail value` -> `e.detail - mousedown`)

Reply:
Done. Tests structure is updated according to browser behavior.